test/e2e_new/e2e_suite_test.go

```diff
@@ -144,6 +184,10 @@ var _ = SynchronizedAfterSuite(func() {
 	By("Tearing down the management cluster")
 	if !skipCleanup {
 		tearDown(bootstrapClusterProvider, bootstrapClusterProxy)
```

Comment:
This should also likely clean up `sshPrivateKey` and `bootstrapAccessKey` prior to attempting to delete the cloud formation stack, it'll be more important if testing locally and choosing not to delete the cloud formation stack in between.

Reply:
The SSH key creation is idempotent, so still having the SSH key there doesn't affect the run. Maybe the tester uploads their own key before hand, and in that case we don't want to delete it.
As the test doesn't actually use the private key material, am removing that section.